bin/jobsub_cmd

```diff
@@ -230,6 +230,11 @@ def main() -> None:
             )
             sys.stdout.flush()
 
+            # pipe our remaining output through sort (by date) and jobsub_totals
+            totalsf = os.popen("sort -k 3,4 | jobsub_totals", "w")
```

Comment:
Since we assume that `jobsub_cmd` is in the same directory as `jobsub_totals`, is it reasonable to hardcode the relative path here?  I only ask because my testing of this doesn't work because without installing the RPM, `jobsub_cmd` can't find `jobsub_totals` (since it's looking in `/opt/jobsub_lite/bin`).  I'm thinking something like:

```python
jobsub_totals_path = os.path.join(os.path.dirname(__file__), "jobsub_totals")
# ....
totalsf = os.popen(f"sort -k 3,4 | {jobsub_totals_path}", "w")
```

Something like that, roughly?  What do you think?